### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by easherma**
adding note about the address_dedupe service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +62,8 @@ Like admin lookup, this process slows down the importer quite a bit, and
 requires a large amount of disk space (it uses a LevelDB store to track which
 addresses it has seen), so it defaults to off.
 
+
```
Comment (orangejulius) [inline README.md]: Hey! No extra newlines please. That's two whole bytes wasted.

**Comment by orangejulius** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (easherma) [inline README.md]: :-o

**Comment by missinglink** [comment]
:+1:

**Pull request merged**